frontend/src/theme.ts

```diff
@@ -31,11 +31,39 @@ const theme = createTheme({
       defaultProps: {
         disableElevation: true,
         variant: 'contained',
+        disableRipple: true,
```

Comment:
No need to set `disableRipple` prop here since you've set it to `MuiButtonBase`.

How can I disable the ripple effect globally?
https://mui.com/material-ui/getting-started/faq/#how-can-i-disable-the-ripple-effect-globally

Reply:
Good catch